beebop/assignClusters.py

```diff
@@ -102,25 +102,47 @@ def get_clusters(
             queries_clusters,
         )
     else:
-        result = assign_clusters_to_result(
-            zip(queries_names, queries_clusters)
-        )
+        result = get_internal_clusters_result(queries_names, queries_clusters)
 
     save_result(config, result)
     return result
 
 
+def get_internal_clusters_result(
+    queries_names: list[str], queries_clusters: list[str]
+) -> dict:
+    """
+    [Get internal clusters result]
+
+    :param queries_names: [list of sample hashes]
+    :param queries_clusters: [list of sample PopPUNK clusters]
+    :return dict: [dict with index (key)
+        and sample hash with cluster and raw cluster number (value)]
+    """
+    return assign_clusters_to_result(
+        zip(
+            queries_names,
+            [
+                {"cluster": cluster, "raw_cluster_num": cluster}
```

Comment:
Do you actually need both "cluster" and "raw_cluster_num"? They're being assigned the same value.